app/ui/src/app/customizations/api-connector/api-connector.model.ts

```diff
@@ -0,0 +1,36 @@
+import { BaseEntity } from '../../model';
+
+export interface ApiConnector extends BaseEntity {
+  kind: string;
+  data: ApiConnectorData;
+}
+export type ApiConnectors = Array<ApiConnector>;
+
+export interface ApiConnectorData extends BaseEntity {
+  id: string;
+  name: string;
+  description: string;
+  icon: string;
+  properties: ApiConnectorProperties;
+  connectorProperties: {};
+}
+
+export interface ApiConnectorProperties extends BaseEntity {
+  specification: {
+    kind: string;
+    displayName: string;
+    required: boolean;
+    type: string;
+    javaType: string;
+    //tags: Array<Tag>;
```

Comment:
If you do not plan to use this commented property now, just remove it. Pushing commented out code is usually considered a source for technical debt for the future

Reply:
We will need it, but perhaps not for this sprint. I'll just remove it.